addons/website_event/security/ir.model.access.csv

```diff
@@ -17,13 +17,3 @@ access_website_event_menu_portal,website.event.menu,model_website_event_menu,bas
 access_website_event_menu_employee,website.event.menu,model_website_event_menu,base.group_user,1,0,0,0
 access_website_event_menu_user,website.event.menu.user,model_website_event_menu,event.group_event_user,1,1,1,1
 access_website_visitor_user,website.visitor.user,model_website_visitor,event.group_event_user,1,1,0,0
-access_event_question_public,event.question,model_event_question,base.group_public,1,0,0,0
```

Comment:
As explained before, would keep most "public" rules here, only event groups should get access to models I suppose.